### PR TITLE
fix: make attachment API more similar to dict API

### PR DESCRIPTION
**Pull request by wyfo**
relates to https://github.com/eclipse-uprotocol/up-client-zenoh-python/pull/3

### CODE REVIEW

**Comment by wyfo** [comment]
@Mallets @evshary

File: tests/test_session.py [hunk 1]
```diff
@@ -160,7 +171,7 @@ def callback(sample: Sample):
 def test_session():
     zenoh.init_logger()
     (peer01, peer02) = open_session(["tcp/127.0.0.1:17447"])
-    run_session_qryrep(peer01, peer02)
-    run_session_pubsub(peer01, peer02)
+    # run_session_qryrep(peer01, peer02)
```
Comment (evshary) [inline tests/test_session.py:174]: I guess you forgot to put the tests back

**Review by evshary — CHANGES_REQUESTED** [review]

File: tests/test_session.py (hunk 1 above)
Comment (wyfo) [reply tests/test_session.py:174]: Oups 😅

**Review by evshary — APPROVED** [review]
LGTM

**Review by Mallets — APPROVED** [review]

**Review by Mallets — APPROVED** [review]

**Pull request merged**